context/app/routes_browse.py

```diff
@@ -139,18 +139,12 @@ def details_rui_json(type, uuid):
         abort(404)
     client = get_client()
     entity = client.get_entity(uuid)
-    # For Samples...
+    # For samples and datasets, the nearest RUI location is indexed with the entity itself.
+    # https://github.com/hubmapconsortium/search-api/pull/860
     if 'rui_location' in entity:
         return json.loads(entity['rui_location'])
```

Comment:
Datasets don't include an `rui_location` unless you've made some corresponding change in the `search-api`.

Reply:
I did make this change: https://github.com/hubmapconsortium/search-api/blob/dev-integrate/src/hubmap_translation/addl_index_transformations/portal/translate.py#L307